src/main/java/com/connectfour/Presenter.java

```diff
@@ -1,23 +1,17 @@
 package com.connectfour;
 
 public class Presenter {
-    public static final String ANSI_RESET = "\u001B[0m";
-    public static final String ANSI_BLACK = "\u001B[30m";
-    public static final String ANSI_RED = "\u001B[31m";
-    public static final String ANSI_GREEN = "\u001B[32m";
-    public static final String ANSI_YELLOW = "\u001B[33m";
-    public static final String ANSI_BLUE = "\u001B[34m";
-    public static final String ANSI_PURPLE = "\u001B[35m";
-    public static final String ANSI_CYAN = "\u001B[36m";
-    public static final String ANSI_WHITE = "\u001B[37m";
 
+    public static final String ANSI_BLUE = "\u001B[34m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_RESET = "\u001B[0m";
 
-    public static void printBoard(Displayable display, Board board){
+    public static void printBoard(Displayable display, Boardable board){
         Discable disc;
 
-        for(int i = 0; i < board.grid.length; i++){
-            for(int j = 0; j < board.grid[0].length; j++){
-                disc = board.grid[i][j];
+        for(int i = 0; i < board.getGrid().length; i++){
+            for(int j = 0; j < board.getRow(0).length; j++){
+                disc = board.findDisc(i,j);
```

Comment:
I like that this method is pretty flexible and knows just enough about the inner workings of the board in order to display it :)